libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRoute.java

```diff
@@ -585,15 +602,21 @@ public Builder routeOptions(RouteOptions options) {
         directionsBuilder.addApproaches(approaches);
       }
 
+      String separatesLegs = options.separatesLegs();
+      if (!TextUtils.isEmpty(separatesLegs)) {
+        Integer[] splitSeparatesLegsIndices = parseSeparatesLegsIndices(separatesLegs);
+        directionsBuilder.addSeparatesLegs(splitSeparatesLegsIndices);
+      }
+
       if (!TextUtils.isEmpty(options.waypointNames())) {
         String[] waypointNames = options.waypointNames().split(SEMICOLON);
         directionsBuilder.addWaypointNames(waypointNames);
       }
 
       String waypointTargets = options.waypointTargets();
       if (!TextUtils.isEmpty(waypointTargets)) {
-        Point[] splittedWaypointTargets = parseWaypointTargets(waypointTargets);
-        directionsBuilder.addWaypointTargets(splittedWaypointTargets);
+        Point[] splitdWaypointTargets = parseWaypointTargets(waypointTargets);
```

Comment:
NIT `splitd` is not a word: `split` I think is what you're looking for

Reply:
![troll](https://user-images.githubusercontent.com/1668582/52641655-59a65580-2ea7-11e9-995c-e83101201195.gif)

BTW, good catch! Will fix 👍